day_two_guide/topics/ntp_synchronization.adoc

```diff
@@ -0,0 +1,49 @@
+////
+NTP Synchronization
+
+Module included in the following assemblies:
+
+* day_two_guide/run_once_tasks.adoc
+////
+
+NTP (Network Time Protocol) is for keeping hosts in sync with the world clock.
+Time synchronization is important for time sensitive operations, such as log
+keeping and time stamps, and is highly recommended for Kuberbetes, which
+{product-title} is built on. {product-title} operations include etcd leader
+election, health checks for pods and some other issues, and helps prevent time
+skew problems.
+
+Depending on your instance, NTP may not be enabled by default. To verify that a host is configured to use NTP:
+
+----
+$ timedatectl
+      Local time: Thu 2017-12-21 14:58:34 UTC
+  Universal time: Thu 2017-12-21 14:58:34 UTC
+        RTC time: Thu 2017-12-21 14:58:34
+       Time zone: Etc/UTC (UTC, +0000)
+     NTP enabled: yes
+NTP synchronized: yes
+ RTC in local TZ: no
+      DST active: n/a
+----
+
+If both `NTP enabled` and `NTP synchronized` are `yes`, then NTP synchronization
+is active. 
+
+If `no`, install the `ntp` rpm package:
```

Comment:
If \`no\`, the instance can be configured to use NTP by installing the \`ntp\` or \`chrony\` rpm packages and enabling it:

\----
\# yum install chrony -y
\# systemctl enable chronyd --now
\# timedatectl set-ntp true
\----